modules/core/src/test/scala/fs2/kafka/CommittableConsumerRecordSpec.scala

```diff
@@ -21,7 +21,11 @@ final class CommittableConsumerRecordSpec extends BaseSpec {
 
       assert {
         record.toString == "CommittableConsumerRecord(ConsumerRecord(topic = topic, partition = 0, offset = 0, key = key, value = value), CommittableOffset(topic-0 -> 0, the-group))" &&
-        record.show == "CommittableConsumerRecord(ConsumerRecord(topic = topic, partition = 0, offset = 0, key = key, value = value), CommittableOffset(topic-0 -> 0, the-group))"
+        record.show == "CommittableConsumerRecord(ConsumerRecord(topic = topic, partition = 0, offset = 0, key = key, value = value), CommittableOffset(topic-0 -> 0, the-group))" &&
+        (record.record match {
+          case ConsumerRecord("topic", 0, 0L, "key", "value", _) => true
+          case _                                                 => false
+        })
```

Comment:
I think this part could be moved into its own test as its unrelated to Show and toString